includes/course-theme/class-sensei-course-theme.php

```diff
@@ -209,26 +208,6 @@ public function get_wrapper_template() {
 		return locate_template( 'index.php' );
 	}
 
-	/**
-	 * It overrides the template content, loading the respective
-	 * template and rendering the blocks from the template.
-	 *
-	 * @access private
-	 *
-	 * @return string The content with template and rendered blocks.
-	 */
-	public function override_template_content() {
```

Comment:
Started moving some of this theme-specific logic into the theme's `functions.php`.